<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>主菜单</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>391</width>
      <height>211</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>391</width>
      <height>211</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>281</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>71</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Umpush</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>Ciphertext Search System</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>320</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>关键字：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>236</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>236</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="do_search_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>89</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>检索</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>351</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>351</width>
        <height>71</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Rekha</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Rekha'; font-size:11pt; font-weight:600; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:400; font-style:normal;&quot;&gt;返回文档数   TOP-K   关键词1 关键词2 关键词3  ...&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-weight:400; font-style:normal;&quot;&gt;例 ：     3	       3                aa           ab           ac&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="third_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>341</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>全部下载</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionaction_3"/>
    <addaction name="actionaction_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>管理员</string>
    </property>
    <addaction name="actionaction_1"/>
    <addaction name="actionaction_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_5"/>
   <addaction name="action_7"/>
   <addaction name="action_6"/>
  </widget>
  <action name="actionaction_1">
   <property name="text">
    <string>生成数据</string>
   </property>
  </action>
  <action name="actionaction_2">
   <property name="text">
    <string>生成密钥</string>
   </property>
  </action>
  <action name="actionaction_3">
   <property name="text">
    <string>账户注销</string>
   </property>
  </action>
  <action name="actionaction_4">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>上传</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>下载</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>检索</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>关于本系统</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>系统帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
